README.md

```diff
@@ -27,15 +27,19 @@ You can install QIT in three different ways:
 
 ### Globally Using `wget`
 
-_(Pro Tip: Opting for the Composer installation method simplifies the process of updating QIT in the future 😉)_
+_(Pro Tip: Opting for the Composer installation method simplifies the process of [updating QIT](https://woocommerce.github.io/qit-documentation/#/cli/getting-started?id=updating-qit) in the future 😉)_
 1. Run `wget https://github.com/woocommerce/qit-cli/raw/trunk/qit`
 2. Execute `chmod +x qit`
 3. Move the file to a directory in your PATH, such as `sudo mv qit /usr/local/bin/qit`
 4. Run `qit` to authenticate with your WooCommerce.com Partner Developer account.
 
-## Documentation
+## Examples
 
-For more detailed information on QIT and how to use it, refer to the [official documentation](https://woocommerce.github.io/qit-documentation/#/).
+- `qit run:e2e my-extension` - Runs the WooCommerce Core E2E tests with your extension active.
+- `qit run:e2e my-extension --php_version=8.2` - Same as above, but with PHP 8.2.
+- `qit run:e2e my-extension --php_version=8.2 --woocommerce_version=rc --wordpress_version=rc` - Same as above, but with PHP 8.2, and the release candidate versions of WooCommerce and WordPress, if they are available.
+- `qit run:e2e my-extension --zip` - Runs the tests using your development build zip, that is still unreleased.
```

Comment:
Debating on tweaking the wording here a bit, what do you think?

```suggestion
- `qit run:e2e my-extension --zip` - Runs the tests using an unreleased development build zip.
```

Reply:
Thanks for the suggestion, tweaked